client/src/views/home/HomePage.vue

```diff
@@ -352,7 +355,14 @@ async function login(device: AvailableDevice, password: string): Promise<void> {
   if (result.ok) {
     router.push({ name: 'workspaces', params: {handle: result.value}});
   } else {
-    console.log('Could not login', result.error);
+    const notif = new Notification(
```

Comment:
Is there a way to have default values for arguments? Or named arguments?
I have no idea what `false` and `{}` do in that function call. I would like to be able to do either
`new Notification('message', 'title', NotificationLevel.Error)`
and/or
`new Notification('message', title='title', level=NotificationLevel.Error)`

Reply:
Add default values for data and trace